src/utils/index.ts

```diff
@@ -0,0 +1,91 @@
+import { ethers, ContractInterface, ContractTransaction, Signer } from 'ethers';
+import { Provider } from '@ethersproject/abstract-provider';
+import { API } from 'bnc-notify';
+import BigNumber from 'utils/bignumber';
+import ERC20ABI from 'abis/ERC20.json';
+import TokenGeyserABI from 'abis/TokenGeyser.json';
+
+export const bnToDec = (bn: BigNumber, decimals = 18): number => {
+  return bn.dividedBy(new BigNumber(10).pow(decimals)).toNumber();
+};
+
+export const decToBn = (dec: number, decimals = 18): BigNumber => {
+  return new BigNumber(dec).multipliedBy(new BigNumber(10).pow(decimals));
+};
```

Comment:
I'm a bit nervous of converting `BN`s to numbers... I'd rather keep them as strings, as ethereum numbers are big enough to make Javascript numbers [overflow](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_VALUE).

Reply:
I guess what would be a better way of displaying `bigint` as a decimal on the frontend? One thought is doing something like:

```
export const bigintToDec = (bigint: bigint): number => {
  const convertedNumber = Number(bigint / BigInt(1e9));
  return convertedNumber / 1e9;
};
```

Reply:
To display them, we can just convert them to String and use [token-amount](https://github.com/aragon/token-amount/) to format them correctly. :)

Reply:
Fixed in 6cd632d7c520d20ea5f02a83b1ea1b4da83ad696